### PR TITLE
Anton/fix_dev_branch

### PR DESCRIPTION
**Pull request by romanukhaav**
Roma & Serg check please working of project

### CODE REVIEW

File: src/test/java/rv/atqc/gttesting/archex/PlacesPage.java [hunk 1]
```diff
@@ -67,17 +65,15 @@
     private WebElement textSearchResults;    
     @FindBy(how = How.XPATH, using = "//*[@id=\"57a4d84dca7a727c0ca59ba3\"]/button/h3/a")
     private WebElement foundResultForVisibleItem;
-
-    @FindBy(how = How.XPATH, using = "//*[@id=\"584e7ef0b61f280400d36ebd\"]/button/h3/a")
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/archex/PlacesPage.java:71]: Are you sure you can delete this?

**Review by dmatsev — APPROVED** [review]
with question

File: src/test/java/rv/atqc/gttesting/archex/PlacesPage.java (hunk 1 above)
Comment (romanukhaav) [reply src/test/java/rv/atqc/gttesting/archex/PlacesPage.java:71]: Yes. This is pageobject from my userstory. And i don`t use this parameter.

**Review by MelnykNikita — APPROVED** [review]
my tests are runnable.

**Review by TyshchenkoRoma — APPROVED** [review]

**Pull request merged**